time stamp
float64[] target_vel
float64[] current_vel
float64[] target_pos
float64[] current_pos